### ISSUE TITLE
Github Issues Page

### ISSUE THREAD

**Issue by saravanabalagi** [DarkNightMode/Dark-Night-Mode-Chrome-Extension#31]
Glitch

![image](https://user-images.githubusercontent.com/8567893/63063445-9e618680-bef4-11e9-9d36-08cabdb33858.png)

**Comment by saravanabalagi** [comment]
It's here

![image](https://user-images.githubusercontent.com/8567893/63063643-5b53e300-bef5-11e9-8dfe-723d4ad81c2f.png)

**Comment by DjSni** [comment]
i have also found a glitch, marked with the red arrows:

![Screenshot_116](https://user-images.githubusercontent.com/31524614/67620152-d5f98280-f803-11e9-980e-ea91b2980203.png)

for information, my used browser:
Vivaldi 2.8.1664.44 (Stable channel) (64-Bit) & this extension on version 2.0.5 from Chrome store

**Comment by iarigby** [comment]
Same is happening for me, text is impossible to read

**Comment by saravanabalagi** [comment]
Yep, in the older versions, the text was visible (shown in my screenshot), but now it's just grey as uploaded by @DjSni. Perhaps the update caused it? Some instructions on how to upload those rules from the contributors would be great.

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/17880848/73910678-073f1f80-48d6-11ea-8562-b5fff4cc2280.png)

![image](https://user-images.githubusercontent.com/17880848/73910744-2ccc2900-48d6-11ea-8373-dd5e40b7d595.png)

![image](https://user-images.githubusercontent.com/17880848/73910846-7d438680-48d6-11ea-8dcf-6beeab838ac9.png)
